content/login/LoginController.js

```diff
@@ -95,6 +98,10 @@ function($scope, $routeParams, loginStatusProvider, eventHandler, pageService, u
 	$scope.AdminButtonClick = function() {
 		pageService.NavigateToPage('admin');
 	};
+	$scope.PublicProfileButtonClick = function() {
+		let userLink = $scope.userLink.link;
```

Comment:
We aren't using babel, so unfortunately this has to be `var`, `let` isn't supported until es6.